city_scrapers/spiders/cook_justice_advisory.py

```diff
@@ -0,0 +1,196 @@
+
+    def _parse_event(self, response):
+        """Parse the event page."""
+        title = self._parse_title(response)
+        start = self._parse_start(response)
+        links_key = start.strftime("%y-%m")
+
+        meeting = Meeting(
+            title=title,
+            description=self._parse_description(response),
+            classification=ADVISORY_COMMITTEE,
+            start=start,
+            end=self._parse_end(response),
+            time_notes="",
+            all_day=self._parse_all_day(response),
+            location=self._parse_location(response),
+            links=self.agenda_map[links_key],
+            source=response.url,
+        )
+
+        meeting["id"] = self._get_id(meeting)
+        meeting["status"] = self._get_status(meeting)
+
+        return meeting
+
+    def _get_event_urls(self, response):
+        """
+        Get urls for all justice advisory council (JAC in calendar) meetings on the page
+        """
+        responses = []
+        events = response.xpath("//a[contains(@href, 'event')]")
+        for event in events:
+            event_title = event.xpath("text()").extract_first().lower()
+            if (
+                "jac" in event_title
+                or "justice advisory" in event_title
+                or ("justice" in event_title and "advisory" in event_title)
+            ):
+                href = event.xpath("./@href").extract_first()
+                responses.append(response.urljoin(href))
+        return responses
+
+        """
+        return [
+            response.urljoin(href)
+            for href in response.xpath(
+                '//a[contains(text(), "JAC")]|//a[contains(text(), "Justice Advisory")]'
+            )
+            .css("a::attr(href)")
+            .extract()
+        ]
+        """
+
+    def _parse_location(self, response):
+        """
+        Parse or generate location.
+        """
+        address = response.xpath(
+            '//div[@class="field event-location"]/descendant::*/text()'
+        ).extract()
+        for word in ["Location:", ", ", " "]:
+            address.remove(word)
+        address = " ".join(address)
+        if "Microsoft Teams" in address:
+            return {
+                "address": "",
+                "name": "",
+            }
+        else:
+            return {
+                "address": address,
+                "name": "",
+            }
+
+    def _parse_all_day(self, response):
+        """
+        Parse or generate all-day status. Defaults to false.
+        """
+        date = response.xpath(
+            '//span[@class="date-display-single"]/descendant-or-self::*/text()'
+        ).extract()
+        date = "".join(date).upper()
+        return "ALL DAY" in date
+
+    def _parse_title(self, response):
+        """Parse or generate event"""
+        title = "".join(response.css("h1::text").extract())
+        if "JAC Council Meeting" in title:
+            return "JAC Council Meeting"
```

Comment:
I should have caught this before, but this can be "Justice Advisory Council" and we can drop "Meeting"